test/apps/iodemo/io_demo.cc

```diff
@@ -2664,6 +2669,38 @@ static void adjust_opts(options_t *test_opts) {
         test_opts->chunk_size = std::min(test_opts->chunk_size,
                                          test_opts->max_data_size);
     }
+
+    if (servers.empty()) {
+        return;
+    }
+
+    for (auto server = servers.cbegin(); server != servers.cend(); ++server) {
+        repeat_separator = strchr(*server, ':');
+        if (repeat_separator == NULL) {
+            test_opts->servers.push_back(std::string(*server));
```

Comment:
can remove std::string it will use the constructor automatically

Reply:
Done